test/test-patientsController.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable func-names */
+const app = require('../index')
+const chai = require('chai')
+const sinon = require('sinon')
+const clean = require('./helper/clean')
+const dbPatients = require('../db/patients')
+
+describe('patientsController', function() {
+  describe('create patient', function() {
+    let insertPatientStub;
+    beforeEach(async function(done) {
+      insertPatientStub = sinon.stub(dbPatients, 'insertPatient')
+        .returns(Promise.resolve([ {
+          'firstNames': 'prenom',
+          'lastName': 'nom',
+          'INE': 'studentNumber'}
+        ]))
+      done()
+    })
+
+    afterEach(async function() {
+      await clean.cleanAllPatients()
```

Comment:
on utilise pas la vraie db, donc on a pas besoin de clean.
Par contre il faut faire `insertPatientStub.restore()` sinon le test suivant va planter (ici ca se voit pas parce qu'il y a un seul test !)

```suggestion
      insertPatientStub.restore()
```